templates/api-server.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "api-server",  
+  "description": "A LoopBack API server with local User auth",
+  "package": {
+    "version": "1.0.0",
+    "main": "server/server.js",
+    "scripts": {
+      "lint": "eslint .",
+      "start": "node .",
```

Comment:
we should consider including `"test": "mocha"` in the default templates -- @bajtos thoughts? I feel it's weird how we can scaffold a LB app and not be able to run `npm test` OOB (since we're opinionated on other things like `eslint` anyways).

@deepakrkris This is outside the scope of this PR of course.